qt/draw_widget.cpp

```diff
@@ -703,7 +703,7 @@ void DrawWidget::SetDefaultSurfaceFormat(bool apiOpenGLES3)
   else
   {
     fmt.setProfile(QSurfaceFormat::CompatibilityProfile);
-    fmt.setVersion(2, 1);
+    fmt.setVersion(2, 0);
```

Comment:
Why?

Reply:
Nothing relies on `2, 1` in a desktop GL scenario, and AFAIK the proper way of requesting ES 2.0 is this.
And all this `setVersion` call is just a polite request anyway, which might or might not be abided.
So it's not super relevant anyway.
If it is a big eye soar, then I can re-test with `2, 0`, I just don't see much value in it.

Reply:
Why it was chosen in the first place instead of 2.0? Was there some reasoning in commit/pr for mapsme/omim repo?

Reply:
From the git history It is unclear to me. It was simply selected in https://github.com/organicmaps/organicmaps/commit/3434dd895e5cac2ef9167b3118a5b006072ab9c1,
for unknown reasons, even though the commit was not even about ES2, but desktop OpenGL 3.
And previous to that commit there was no call to `QSurfaceFormat::setVersion` in the codebase whatsoever.

Reply:
And if I may add my commentary, to https://github.com/organicmaps/organicmaps/commit/3434dd895e5cac2ef9167b3118a5b006072ab9c1, then there it  already appears to be inconsistent with the shader codes IMHO. Becase both the fragment and the vertex shaders were added without version directive, which AFAIK had to default to `#version 110`, and that does not correspond to `2, 1` but to `2. 0`, if I am not mistaken.

Reply:
The old code was working with the desktop OpenGL 2.1 and then (after the mentioned patch) with 3.0. ES is not related here.

Requesting (non-ES) OpenGL 2.1 instead of 2.0 is a good idea.